projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -515,6 +533,16 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
 
   private onMouseMove(): void {
     const locationData = this.getMouseDataForCurrentEvent();
+
+    // tslint:disable-next-line
+    if (this.groupId) {
```

Comment:
group id should only be used when deciding how to interact with the sync service, not at the event level itself (a hover is a hover, whether it belongs to a group or not).